courses/intro-to-ada/chapters/modular_programming.rst

```diff
@@ -237,3 +239,434 @@ This example shows how :ada:`Last_Increment` is used indirectly:
        R := Increment_By (I);
        Display_Update_Values;
     end Main;
+
+Child and nested packages
+-------------------------
+
+Packages can be used to create hierarchies. We achieve this by using child
+or nested packages, which are both subunits of a parent package. One example
```

Comment:
I think saying "are both subunits" is not clear - internal and external visibility issues are very different. In my opinion, child packages generally extend the capability of the parent, whereas nested packages generally modularize the enclosing package. Your wording implies they could be used similarly